clash-prelude/clash-prelude.cabal

```diff
@@ -317,14 +319,16 @@ test-suite unittests
       base,
       hint          >= 0.7      && < 0.10,
       tasty         >= 1.2      && < 1.3,
-      tasty-hunit
+      tasty-hunit ,
+      template-haskell  >= 2.9.0.0 && < 2.14
```

Comment:
This should be `template-haskell  >= 2.9.0.0 && < 2.16` to work with GHC-8.8

Reply:
Actually you could just remove the bounds here and cabal will use the bounds places on the `Library`